indexd/drs/blueprint.py

```diff
@@ -0,0 +1,153 @@
+import flask
+
+from indexd.blueprint import dist_get_record
+
+from indexd.errors import AuthError
+from indexd.errors import UserError
+from indexd.index.errors import NoRecordFound as IndexNoRecordFound
+from indexd.drs.errors import UnexpectedError
+
+from indexd.fence_client import get_signed_url_for_object, FenceRequest
+
+try:
+    from local_settings import settings
+except ImportError:
+    from indexd.default_settings import settings
+
+import requests
+
+blueprint = flask.Blueprint("drs", __name__)
+
+blueprint.config = dict()
+blueprint.index_driver = None
+blueprint.dist = []
+
+@blueprint.route("/ga4gh/drs/v1/objects/<path:object_id>", methods=["GET"])
+def get_drs_object(object_id):
+    """
+    Returns a specific DRSobject with object_id
+    """
+    ret = blueprint.index_driver.get(object_id)
+    return flask.jsonify(indexd_to_drs(ret)), 200
+
+    
+@blueprint.route("/ga4gh/drs/v1/objects", methods=["GET"])
+def list_drs_records():
```

Comment:
TODO see if we can just call the existing list function from "/index"
and call `indexd_to_drs` on each record